narayana-release-process.sh

```diff
@@ -175,7 +181,7 @@ else
 fi
 
 rm -rf $PWD/localm2repo
-./build.sh clean install -Dmaven.repo.local=${PWD}/localm2repo -DskipTests -gs ~/.m2/settings.xml -Dorson.jar.location=$ORSON_PATH -Pcommunity
+./build.sh clean install -Dmaven.repo.local=${PWD}/localm2repo -DskipTests -gs ~/.m2/settings.xml -Dorson.jar.location=$ORSON_PATH -Pcommunity -Pnative-docker $CONTAINER_BUILD_RUNTIME
```

Comment:
I was chatting with @ochaloup I think that if the image that is built also contains a platform dependent native image that will need to be reflected in the docker image name that we push: https://github.com/jbosstm/narayana/blob/f3a0bbb5784b21d1d097311d1e15b4028b64679d/narayana-release-process.sh#L228